### ISSUE TITLE
A11Y Documentation

### ISSUE THREAD

**Issue by irmerk** [accordproject/techdocs#70]
Accord Project documentation needs to be updated to represent our ongoing effort for inclusive design for accessibility. This involves:

**Inclusive design** is a methodology which enables and draws on the full range of human diversity.

**Accessibility** is qualities which make an experience open to all and the professional discipline aimed at achieving that.

Good resources to share: 
- [The A11Y Project][a11yproject] 
- [WCAG][wcag]
  
This will encompass [cicero-ui][cicero-ui] and [template-studio-v2][template-studio-v2]
  
[a11yproject]: https://a11yproject.com/
[wcag]: https://www.w3.org/TR/WCAG20/
[cicero-ui]: https://github.com/accordproject/cicero-ui
[template-studio-v2]: https://github.com/accordproject/template-studio-v2

**Comment by irmerk** [comment]
### Hacktoberfest:
This issue can provide for multiple pull requests.

**Comment by irmerk** [comment]
@AninditaBasu unsure exactly, was looking for community input on how this could take shape in our documentation.

**Comment by hp-plus5** [comment]
I'll throw out some topics to cover. I'd love some feedback on these; I'm not well-integrated into this community, so there are some questions and assumptions here that I don't know the value of (or if they're relevant).

- Overall accessibility concepts as defined by X standards (such as The A11y Project, WCAG, or any global legislation that declares necessary accessibility (whether technically specific or not))

- What efforts have been made in Accord's past (especially within cicero-ui and template-studio-v2) to implement inclusive design, and what incentives (if any) have been added to foster participation

- Primary contributors towards accessibility overall, especially in helping bring the issue to light and begin building active momentum

- What are the base requirements as defined by Accord in order to ensure new features are accessible. As technical as possible without creating content that will be almost immediately out of date; it would be nice if this could be referenced as a guide to newcomers wanting to make pull requests.

- How to test accessibility; information about screenreaders used by Accord to help manually test code when required

**Comment by irmerk** [comment]
@hp-plus5 yes these are great topics. I would say that we aim to be compliant with [WCAG 2.1](https://www.w3.org/TR/WCAG21/), considering it seems to be the most widespread.

I'm going to turn your points into a checklist in the opening post for this issue to follow.

**Comment by skingh** [comment]
I want to contribute on this!

**Comment by CH-ND-N** [comment]
Hey @irmerk , Please assign this to me .

**Comment by CH-ND-N** [comment]
thanks @irmerk, for assigning me this project. I need some guidance please help me get started.

**Comment by irmerk** [comment]
Sure @CH-ND-N. I think a good start would be accumulating information that is applicable to AP in regards to accessibility and creating a PR for a section within https://docs.accordproject.org/docs/ref-web-components-overview.html. Using the material in the OP of this Issue would be helpful I think.

**Comment by CH-ND-N** [comment]
Thanks, I am on it.

**Comment by Dronten** [comment]
can i work on this issue , I will require a guidance , a little

**Comment by Gurmeet-byte** [comment]
Hi, I’d like to contribute by adding a short documentation section explaining WCAG 2.1 and its relevance to Accord Project.
Please let me know if that sounds good.

**Comment by Anushka-002** [comment]
Hi, I’d like to work on this issue as part of my initial contributions.

I’m planning to create a structured accessibility and inclusive design guide, covering WCAG 2.1 principles, testing methods, screen reader usage, and some useful resources for contributors.

Please let me know if this approach works or if there are any specific guidelines I should follow before starting.

Thank you!
